<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>453</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cb3DItemType">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>LinePlotItem</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>ScatterPlotItem</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="cbAntialias">
       <property name="text">
        <string>Antialias</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>1</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="page3DLinePlotItem">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QgsColorButton" name="btn3DLinePlotItemColor">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>16</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="colorDialogTitle">
          <string>Set line color</string>
         </property>
         <property name="color">
          <color>
           <red>255</red>
           <green>0</green>
           <blue>0</blue>
          </color>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <widget class="QComboBox" name="cb3DLinePlotItemMode">
         <item>
          <property name="text">
           <string>lines</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>line_strip</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Mode</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_14">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="sb3DLinePlotItemWidth"/>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Line width</string>
         </property>
        </widget>
       </item>
       <item row="0" column="6">
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page3DScatterPlotItem">
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="3">
        <widget class="QDoubleSpinBox" name="sb3DScatterPlotItemSize"/>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Color</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QgsColorButton" name="btn3DScatterPlotItemColor">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>16</height>
          </size>
         </property>
         <property name="colorDialogTitle">
          <string>Set point color</string>
         </property>
         <property name="color">
          <color>
           <red>0</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </property>
        </widget>
       </item>
       <item row="0" column="5">
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="4">
        <widget class="QCheckBox" name="cb3DScatterPlotItemPxMode">
         <property name="text">
          <string>size as pixels</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Point size</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page3DMeshItem">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <property name="spacing">
        <number>6</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QgsColorButton" name="btn3DMeshItemColor">
         <property name="colorDialogTitle">
          <string>Select mesh color</string>
         </property>
         <property name="color">
          <color>
           <red>85</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QgsColorButton" name="btn3DMeshItemEdgeColor">
         <property name="colorDialogTitle">
          <string>Set edge color</string>
         </property>
         <property name="color">
          <color>
           <red>255</red>
           <green>170</green>
           <blue>0</blue>
          </color>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb3DMeshItemDrawEdges">
         <property name="text">
          <string>Edges</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb3DMeshItemDrawFaces">
         <property name="text">
          <string>Faces</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb3DMeshItemSmooth">
         <property name="text">
          <string>Smooth</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cb3DMeshItemNormals">
         <property name="text">
          <string>Normals</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QGraphicsView" name="plotWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cb3DItemType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>stackedWidget</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>17</y>
    </hint>
    <hint type="destinationlabel">
     <x>110</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
